test/enabled.ts

```diff
@@ -0,0 +1,18 @@
+import { createApp } from 'h3'
+import { listen } from 'listhen'
+import { createAuthMiddleware } from '../src'
+
+const app = createApp()
+
+app.useAsync(createAuthMiddleware({
+  enabled: false,
```

Comment:
I'm thinking to rename to `bypass: true` or `disabled: true` since basically we still have middleware and default is true...

Reply:
disabled sounds great! Do you mind updating and merging? (Going to bed now 😅)

Reply:
Sure. I think finally calling it bypass since later can also support `bypass: (req) => boolean`  for dynamic bypass (like localhost, etc)